src/assets/locales/fr.json

```diff
@@ -531,8 +589,19 @@
   "tips.mobile.locomotion": "Bravo ! Pour vous déplacer, pincez avec deux doigts.",
   "tips.mobile.look": "Bienvenue ! 👋 Cliquez et faites glisser pour regarder autour de vous.",
   "tips.streaming": "Diffusion en cours vers le hall d'attente. Pour quitter le mode diffusion, utilisez le menu Plus quand vous aurez terminé.",
+  "token-type.account-token": "Jeton Compte",
+  "token-type.app-token": "Jeton App",
+  "tokens.administrator-privileges-required": "Privilèges administrateur requis",
+  "tokens.button-create-token": "Créer jeton",
+  "tokens.create-tokens-description": "Créer des jetons pour accéder à l'",
+  "tokens.create-tokens-description-hubs-api": "API Hubs",
+  "tokens.no-access-description1": "Vous n'avez pas les privilèges suffisants pour créer des jetons API.",
+  "tokens.no-access-description2": "Si vous pensez que vous devriez avoir accès à cette page, veuillez demander des privillèges supplémentaires à votre administrateur Hubs.",
```

Comment:
"Si vous pensez que vous devriez avoir accès à cette page, veuillez demander des droits supplémentaires à votre administrateur Hubs."

Reply:
changed in https://github.com/mozilla/hubs/pull/5002/commits/dc950bea967dfee2a131b4177f848525a8943054